doc-content/enterprise-only/getting-started-kogito/con-kogito-microservices-on-ocp.adoc

```diff
@@ -0,0 +1,10 @@
+[id="con-kogito-microservices-on-ocp_{context}"]
+= {KOGITO} on {OPENSHIFT}
+
+You can deploy {KOGITO} microservices on {OPENSHIFT} for cloud implementation. In this architecture, {KOGITO} microservices are deployed as OpenShift pods that you can scale up and down individually to provide as few or as many containers as required for a particular service.
+
+To help you deploy your {KOGITO} microservices on OpenShift, {PRODUCT} provides an operator:
+
+* *Red Hat Process Automation Manager {KOGITO} Operator*: An operator that guides you through the deployment process. The operator is based on the https://sdk.operatorframework.io/[Operator SDK] and automates many of the deployment steps for you. For example, when you give the operator a link to the Git repository that contains your application, the operator can automatically configure the components required to build your project from source and deploy the resulting services.
+
+To install the Red Hat Process Automation Manager {KOGITO} Operator in OpenShift web console, go to *Operators* -> *OperatorHub* in the left menu, search for and select *RHPAM Kogito Operator*, and follow the on-screen instructions to install the latest operator version.
```

Comment:
This seems to be repeated in the procedure. Do we need it here?

Reply:
The section that had this information is removed, therefore I'll keep it in this section.